PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -139,13 +142,13 @@ def _tauId8WPMask(pattern,doc):
 
 _variablesMiniV2 = cms.PSet(
     _tauVarsBase,
-    _mvaAntiEVars,
```

Comment:
The _deepTauVars2017v2 is not present in master and is also unused in 10_6, so can be cleaned up

https://github.com/cms-sw/cmssw/pull/33658/files#diff-d54676262d2e5326ee3455e57747fd476b12be16993a8eb0f4794e8771f6526fL123-L129